test/utils/util-test.js

```diff
@@ -20,23 +20,27 @@ describe('util', function() {
 
     it('should create a batch with all domains only if the output count is below permitted limit', function() {
       const limit = 200;
-      const { validDomains, rejectedDomains } = util.createBatch(limit, outputMap);
-      assert.deepStrictEqual(validDomains, [domain1, domain2, domain3, domain4]);
+      const { validDomains, rejectedDomains } = util.createBatch2(limit, outputMap);
```

Comment:
Why is this createBatch2? I can't find that function

Reply:
just realized the mocha is not doing the recurse in to folders, thus i missed this, updating the PR